### ISSUE TITLE
[Help wanted] Building rpm packages 

### ISSUE THREAD

**Issue by corrad1nho** [corrad1nho/qomui#20]
Hey,

I'm looking for someone that could possibly help me building better rpm-packages for qomui. Currently, I'm building these on Arch Linux. The problem I came across recently is that Arch updated Python to version 3.7. which resulted in all python files being moved to /usr/lib/python3.7/site-packages/qomui during the install stage on building the rpm package. On a Fedora installation with Python 3.6. starting Qomui hence fails as the automatically generated console scripts (e.g. /usr/bin/qomui-gui) assume files to be at  /usr/lib/python3.6/site-packages/qomui. This problem does not occur on Debian/Ubuntu as the python package can easily be moved to "/usr/lib/python3/dist-packages".

In short, is it necessary to build different rpms for each version of Python (and if so, can this be automated?)? Or is there are more flexible solution? As for now, Python 3.6 is standard on current Fedora releases but what happens if the next release includes Python 3.7? For the latest release I used a rather ugly hack to get around the problem by adding "--install-purelib=/usr/lib/python3.6/site-packages" to the setup.py install parameters. Have a look at my qomui-spec: 
[qomui.txt](https://github.com/corrad1nho/qomui/files/2283110/qomui.txt)

**Comment by precla** [comment]
Catch the response from:
python -c "import sys; print(sys.path)

The 3rd entry should be the /usr/lib/pythonX.x path.
Now, 'just' include this upon creating/installation.

**Comment by corrad1nho** [comment]
Actually, it doesn't matter what I put there. Those lines can be omitted entirely except the %{!?py3_install: } one.

**Comment by corrad1nho** [comment]
Moved to https://openbuildservice.org/ which allows building rpms for different distributions.
